duckduckgo/Android
# Add support for IPv6 in internal builds only

Task/Issue URL: https://app.asana.com/0/488551667048375/1201928304609012/f

### Description
Add support for IPv6 traffic in AppTP.

See [this task](https://app.asana.com/0/488551667048375/1201928304609012/f) and [this other task](https://app.asana.com/0/488551667048375/1201862327796575) for full context.

### Steps to test this PR
_Best effort test_
* Testing if Ipv6 works is hard because it depends on external factors like the ISP.
* This why we enable it for internal builds only for now. Because we have some internal users that were hitting IPv6 issues and they already verified this improvement should work
* The best you can do is to switch to cellular network
* Filter logcat by `TcpNetworkToDevice`
* And see if you spot any IPv6 address in the logs

_Smoke tests for AppTP_
Repat smoke tests below for both PLAY and INTERNAL builds

- [x] enable AppTP
- [x] open apps that perform tracking attempts
- [x] verify blocking works as usual
- [x] open your most used apps, eg. twitter, insta, mail apps, etc
- [x] verify the continue working as usual
- [x] switch between WIFI and CELLULAR and vice versa
- [x] verify all works as expected, including tracker blocking
- [x] verify both update and fresh install
